modules/ppcp-wc-gateway/services.php

```diff
@@ -2026,4 +2009,44 @@ static function ( ContainerInterface $container ): AuthorizeOrderActionNotice {
 	'button.helper.messages-disclaimers'           => static function ( ContainerInterface $container ): MessagesDisclaimers {
 		return new MessagesDisclaimers();
 	},
+
+	'wcgateway.helper.vaulting-scope'              => static function ( ContainerInterface $container ): bool {
+		try {
+			$token = $container->get( 'api.bearer' )->bearer();
+			return $token->vaulting_available();
+		} catch ( RuntimeException $exception ) {
+			return false;
+		}
+	},
+
+	'button.helper.vaulting-label'                 => static function ( ContainerInterface $container ): string {
+		$vaulting_label = __( 'Enable saved cards and subscription features on your store.', 'woocommerce-paypal-payments' );
+
+		if ( ! $container->get( 'wcgateway.helper.vaulting-scope' ) ) {
+			$vaulting_label .= sprintf(
+				// translators: %1$s and %2$s are the opening and closing of HTML <a> tag.
+				__( ' To use vaulting features, you must %1$senable vaulting on your account%2$s.', 'woocommerce-paypal-payments' ),
+				'<a
+					href="https://docs.woocommerce.com/document/woocommerce-paypal-payments/#enable-vaulting-on-your-live-account"
+					target="_blank"
+				>',
+				'</a>'
+			);
+		}
+
+		$vaulting_label .= '<p class="description">';
+		$vaulting_label .= __( 'This will disable all Pay Later messaging on your site.', 'woocommerce-paypal-payments' );
```

Comment:
@jorgeatorres I understood that this text should be displayed when Vaulting is included in the PayPal account's scope, but I decided to always display it instead to avoid surprises after going through the process of including vaulting in the account.

Like, if having Pay Later is a deal-breaker as a merchant, I think it'd be more convenient to know about its incompatibility with vaulting early on. As always, glad to change this behavior.